deliverables/sprint5/exercise_3_essay.tex

```diff
@@ -0,0 +1,53 @@
+\documentclass[10pt]{article}
+\usepackage[utf8]{inputenc}
+
+\usepackage{geometry}
+\geometry{portrait, margin=1.5in}
+
+\begin{document}
+\title{Space Invaders : Reflection}
+\date{\today}
+\author{\textit{Group 22}\\ \textit{\underline{TI2206 Software Engineering Methods}} \\
+ \\Ege de Bruin \\ Bryan van Wijk \\ Dorian de Koning \\ Jochem Lugtenburg }
+ \maketitle  
+ \begin{center}
+Supervisor: Dr. A. Bacchelli\\
+TA: Danny Plenge\\
+ \end{center}     
+ \begin{center}
+ Delft University of Technology\\
+ Faculty of EEMCS\\
+ \end{center}
+ \thispagestyle{empty}
+ \pagebreak
+
+\section*{Introduction}
+This document contains a reflection on the source code evolution and the route followed to create the final version. In this essay will be looked at how the team worked together and what could been learned from this. It will also reflect on what went well and what could be improved for a next project. To describe how the system is evolved, the first version is compared to the final product of our game. Finally we concluded with what could be learned from the course and the lab project.
+
+\newpage
+\section*{Reflection}
+\subsection*{Requirements}
+In the early stages of the project, selection of the game to implement took place during the first lab hours. After selecting Space Invaders as the game to implement, the team started defining requirements using the MoSCoW method. During the project these requirements were extended, as new features were added to the game. Defining requirements in a clear and ambiguous way helped to distinguish tasks to implement and, without requiring long discussions about how to implement a certain feature. During the following sprints, the requirements were defined after the creation of the sprint plan.
+
+\subsection*{Team}
+As a team, the project was started with five persons. Losing one person in during the first iteration, we had to carry out the work of one extra missing person. Because of this, the team quickly realized we had to maintain a clear organization. Making sure the organization of the project was defined using clear documents and following SCRUM we could implement features relatively fast with minimum overhead. There were also no problems with work that wasn't done or incomplete so we did not have to do work of the previous sprint in the following sprint. This resulted in the fact that the project was constant good on schedule.
```

Comment:
Is it me or is this a bit of a quirky sentence? Also should the we in the next sentence not be removed to match the style of the rest of the report?